Sprint-3/4-stretch-investigate/password-validator.test.js

```diff
@@ -15,12 +15,59 @@ To be valid, a password must:
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
 const isValidPassword = require("./password-validator");
-test("password has at least 5 characters", () => {
-    // Arrange
-    const password = "12345";
-    // Act
-    const result = isValidPassword(password);
-    // Assert
-    expect(result).toEqual(true);
-}
-);
\ No newline at end of file
+
+// test("password has at least 5 characters", () => {
+//     // Arrange
+//     const password = "12345";
+//     // Act
+//    const result = isValidPassword(password);
+//     // Assert
+//     expect(result).toBe(true);
+// }
+// );
+test("password has a unknown character", () => {
+  const password = "Mill3@";
+  const result = isValidPassword(password);
+  expect(result).toBe(false);
+
+})
+
+
+ test("password has at least 5 characters", () => {
+  expect(isValidPassword("12345")).toBe(false);// Fails because no letters or special chars
```

Comment:
When testing a specific requirement (like "at least 5 characters"), the test input should only fail that requirement and pass all others.
If the password "12345" fails because it lacks letters or special characters, not because of its length, it makes the test misleading.

Try testing with this instead since it meets all other requirements _except_ being at least 5 characters:
`Ab1!`